plugins/scaffolder-react/src/next/components/ReviewState/ReviewState.tsx

```diff
@@ -57,6 +57,15 @@ export const ReviewState = (props: ReviewStateProps) => {
           if (definitionInSchema['ui:widget'] === 'password') {
             return [key, '******'];
           }
+
+          if (definitionInSchema.enum && definitionInSchema.enumNames) {
+            return [
+              key,
+              definitionInSchema.enumNames[
```

Comment:
Should we maybe fallback if there's no `enumName` reference to the original value, similar to how I think the dropdown would work in the form?

Reply:
Fallback added to original value if no corresponding element in enumNames with expected index